src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -575,7 +577,11 @@ class PointInCell2DTest : public PointInCellTest<2, ExecSpace>
         "1"                 "\n"
         "1 3 0 1 2 3"     "\n\n"
         "boundary"          "\n"
-        "0"               "\n\n";
+        "4"                 "\n"
```

Comment:
This fixed a bug in windows where mfem was no longer generating boundary elements for these meshes.